app/javascript/styles/_explore.scss

```
@@ -0,0 +1,18 @@
+.study-explore {
+	position: relative;
```

Spacing seems off throughout all CSS properties in this file.

```suggestion
	position: relative;
```